backend/src/main/resources/application.yaml

```diff
@@ -35,13 +35,22 @@ spring:
           keycloak:
             authorization-url: ${keycloak.base-oauth-url}/protocol/openid-connect/auth
             token-uri: ${keycloak.base-oauth-url}/protocol/openid-connect/token
+  datasource:
+    url: jdbc:postgresql:postgres
+    username: postgres
+    password: ${POSTGRES_PASSWORD}
+    database : organizations
+    organization_id : organization_id
+    organization_name : organization_name
+    password_encryption : md5
```

Comment:
Just a small formatting thing, but there shouldn't be a space before the colon in the four above lines.

Reply:
edited formatting